### DISCUSSION TITLE
Changing RAM Usage & Number of Cores to be used in codespaces

### DISCUSSION THREAD

**Discussion by EzraKizito** [community/community#26682] [Codespaces]
<p>How can I change RAM usage and number of cores used in Codespaces? I’m on 2 cores and 4 GB RAM usage right now and I’d like to up that.</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi <a class="mention" href="/ezrakizito">@EzraKizito</a> - apologies for the delay, this post was in our uncategorized so it received less visibility than it would if it were assigned to a category. However, I want to try my best to assist here :slight_smile:</p>
<p>I wanted to introduce you to a recent change that you may not have yet seen: <a href="https://github.community/t/codespaces-beta-expands-to-discussions/165089">Codespaces Beta Expands to Discussions </a></p>
<p>Please take a look for the new public location in order to continue this conversation with our team. Thank you again for being here!</p>
<blockquote>
<p>The Codespaces Beta category has moved over to Discussions! Our Codespaces team is eager to continue in conversations and knowledge-sharing with you as you share your questions and feedback with us. Please head over to the new public space for <a href="https://github.com/github/feedback/discussions/categories/codespaces-feedback">Codespaces Feedback</a> to take a look!</p>
<h4>Resources on Discussions to Get Started</h4>
<ul>
<li><a href="https://docs.github.com/en/discussions">GitHub Discussions Beta</a></li>
<li><a href="https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion">Participating in a discussion</a></li>
</ul>
</blockquote>
